test/core/exe_test.py

```diff
@@ -20,11 +20,18 @@
 import openhtf
 from openhtf import core
 from openhtf import plugs
+from openhtf import PhaseResult
```

Comment:
Other maintainers: Should the tests do the same `import openhtf as htf` as the examples? I think they should